### PR TITLE
Cmd echo

### PR DESCRIPTION
**Pull request by AgnieszkaBylica**
I set new_line_chars='\n' because default is '\n' and '\r' and it changes command output when text contains '\r'.

### CODE REVIEW

**Comment by Ernold11** [comment]
Check List
----------
* [ ]  CI is passed
* [ ]  master is merged into current branch
* [ ]  public methods should be at the top of class, protected at bottom  
       example: `on_new_line()` is public, `_parse()` is protected
* [ ]  Name of methods should tell what its doing, not how are doing  
       example: `_parse_re_via_dev_proto_metric()` -> `_parse_via_dev_proto_metric()`
* [ ]  All parsing lines (on_new_line) should be covered by unit tests
* [ ]  Be sure that  COMMAND_OUTPUT_* / COMMAND_KWARGS_* / COMMAND_RESULT_* cover all parsing lines
* [ ]  Parsed line example, parsing regexp, parsing method should be next to each other
* [ ]  Using named groups inside regexp (like `r"^\s*(?P<ADDRESS>\S+)\s+via\s+(?P<VIA>\S+)”`) is preferred
* [ ]  Longer regexp are before shorter regexp
* [ ]  Resources for pytest should be declared as `@pytest.fixture` or `@pytest.yield_fixture`
* [ ]  There are no sensitive data (company specific) like: IP addr, passwords, commands, tools etc

File: moler/cmd/unix/echo.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        self.current_ret['RESULT'] = list()
+
+    def build_command_string(self):
+        cmd = "echo"
+        if self.options:
+            cmd = "{} {}".format(cmd, self.options)
+        if self.text:
+            cmd = "{} {!r}".format(cmd, self.text)
+        if self.output_file:
+            cmd = "{} > {}".format(cmd, self.output_file)
```
Comment (Ernold11) [inline moler/cmd/unix/echo.py]: When `output_file` parameter is passed into command there is only one mode to write data into file: > which replace all lines in files with new data.
There should be another one write mode: >> which mean append data to existing lines in specific file. 
Please add write_mode parameter into command.

Comment (AgnieszkaBylica) [reply moler/cmd/unix/echo.py]: Can write_mode have a default value ">'?

Comment (Ernold11) [reply moler/cmd/unix/echo.py]: I think yes.

**Comment by marcin-usielski** [comment]
OK

**Pull request merged**